### ISSUE TITLE
flutter_locker: generateDebugProto - Windows support not enabled

### ISSUE THREAD

**Issue by ManmeetChadha9** [infinum/flutter-plugins-locker#28]
Hi,
Seems this library is not supported on Windows.
I continuosly see this error when trying to build my app on a windows machine.

Can someone please check.

Thanks
Manmeet Chadha

[Logs.txt](https://github.com/infinum/flutter-plugins-locker/files/8755105/Logs.txt)

![image](https://user-images.githubusercontent.com/52465483/169833979-69830425-978b-4184-93ac-ebf8f43cb82f.png)

**Comment by tudor07** [comment]
I have the same problem

**Comment by itsJoKr** [comment]
Sorry, we don't have much control of it, because issue comes from protobuf library that we use. Our goal now is to completely replace protobuf, but for that you'll have to wait (hopefully a week or two). I'll keep you updated.

**Comment by ManmeetChadha9** [comment]
Hi Josip... Just wanted to check if you had any update on the issue.

**Comment by Shradha1824** [comment]
Hi Josip... any update on this issue?

**Comment by itsJoKr** [comment]
@Shradha1824 we're testing the new version, so expect it during the next week.

If you want to try it before that you can do it with:
```dart
  flutter_locker:
    git:
      url: https://github.com/infinum/flutter-plugins-locker.git
      ref: 60e72b911f2b40bd081386965b8a7c5db203ef76
```

We have a breaking change: Every parameter in save and retrieve method is now named.

```
FlutterLocker.retrieve(RetrieveSecretRequest(
      key: 'key',
      androidPrompt: AndroidPrompt(
        title: 'title',
        cancelLabel: 'cancel',
        descriptionLabel: 'description',
      ),
      iOsPrompt: IOsPrompt(touchIdText: 'text),
```

**Comment by lukaknezic** [comment]
Hey @ManmeetChadha9. We are now in final stages of testing the lib. Release will probably be early next week 🙂

**Comment by ManmeetChadha9** [comment]
Hey @lukaknezic , any luck with the release ?

**Comment by lukaknezic** [comment]
@ManmeetChadha9 Just published new version with the fix: 2.1.0
It has braking change, because of pigeon all parameters in models are now named parameters instead of unnamed ones

**Comment by tudor07** [comment]
@lukaknezic Looks like `flutter_locker` `2.1.0` requires Flutter 3. Maybe new version should've been `3.0.0`?

**Comment by lukaknezic** [comment]
@tudor07 Yeah you are right, will push a quickfix. For now you can use:

```
  flutter_locker:
    git:
      url: https://github.com/infinum/flutter-plugins-locker.git
      ref: b12db5d9d6ccc9d52c40d049fc76486a2b4c33f1
```

**Comment by lukaknezic** [comment]
@tudor07 Quickfix is available: 2.1.0+1
